api/source/job_control/aggregate_statistics.rst

```diff
@@ -0,0 +1,186 @@
+|                         +------------------+----------------------------------------------------+
+|                         | `Allowed Values` | ["week", "month"]                                  |
+|                         +------------------+----------------------------------------------------+
+|                         | `Example`        | ``group_by=month``                                 |
++-------------------------+------------------+----------------------------------------------------+
+| start_date              | `Description`    | .. raw:: html                                      |
+|                         |                  |                                                    |
+|                         |                  |  Will calculate statistics for jobs returned</br>  |
+|                         |                  |  after the given date. When unspecified,</br>      |
+|                         |                  |  the date of the first returned job is</br>        |
+|                         |                  |  used as the start date.                           |
+|                         |                  |                                                    |
+|                         +------------------+----------------------------------------------------+
+|                         | `Allowed Values` | Date in ISO format.                                |
+|                         +------------------+----------------------------------------------------+
+|                         | `Example`        | ``start_date=2014-08-27T13:40:53``                 |
++-------------------------+------------------+----------------------------------------------------+
+| end_date                | `Description`    | .. raw:: html                                      |
+|                         |                  |                                                    |
+|                         |                  |  Will calculate statistics for jobs returned</br>  |
+|                         |                  |  before the specified date. When unspecified,</br> |
+|                         |                  |  the current time is used as the end date.         |
+|                         |                  |                                                    |
+|                         +------------------+----------------------------------------------------+
+|                         | `Allowed Values` | Date in ISO format.                                |
+|                         +------------------+----------------------------------------------------+
+|                         | `Example`        | ``end_date=2014-08-27T13:40:53``                   |
++-------------------------+------------------+----------------------------------------------------+
+
+- When no range is specified, the response will contain statistics for
```

Comment:
@caitlinrhd Unneeded. Forgot to remove before committing this. start_date/end_time have descriptions of default values when time range is not specified.